### ISSUE TITLE
[Bug] Failed to load AppFlowy UI

### ISSUE THREAD

**Issue by Twilightssuperb** [AppFlowy-IO/AppFlowy#6772]
### Bug Description

After forced PC reboot (because of blackout) I launched previous v.0.4.9 instance from disk C by accident. Got error, closed it. Then I've started **main** v.0.7.3 instance on disk F, but get empty UI.

How to fix it? I've tried to delete all files from AppFlowy instance form disk F and set clear instance, but nothing...

### How to Reproduce

IDK

### Expected Behavior
[log_2024-11-10.txt](https://github.com/user-attachments/files/17723063/log_2024-11-10.txt) (previous normal launch)
[log_2024-11-12.txt](https://github.com/user-attachments/files/17722984/log_2024-11-12.txt)

### Operating System

Win 8.1 x64

### AppFlowy Version(s)

0.7.3

### Screenshots

![image](https://github.com/user-attachments/assets/05c9430c-61e0-4669-b4d8-e4abed9ba658)

**Comment by LucasXu0** [comment]
Hi, @Twilightssuperb. The last version that supports Windows 8 is [v0.4.9](https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.4.9).

Are you using the local mode? And please try to downgrade to v0.4.9.

**Comment by Twilightssuperb** [comment]
I use local mode.

All was OK with v.0.7.3 on this PC before my mistake with accidental v.0.4.9 launch. I attached logs.

Maybe the reason is in some `C:\users\[...]\AppData\Roaming\io.appflowy` files, I don't know. It's not good that AppFlowy isn't completely portable and leaves traces and makes dependencies for standalone instances.

I create third standalone instance for v.0.5.1, but same error again (that for v.0.4.9) — failed to load workspace. Despite no AppFlowy instance is running.
![image](https://github.com/user-attachments/assets/a139eb28-bac7-405b-9bf1-d36b5f89a17a)

**Comment by LucasXu0** [comment]
@Twilightssuperb Can you check if this folder is occupied by another process? In this case, the app can't read any data from the directory, which may cause this error.

```
C:\users\[...]\AppData\Roaming\io.appflowy
```

**Comment by Twilightssuperb** [comment]
> Can you check if this folder is occupied by another process?

Resource Monitor says nothing:
![image](https://github.com/user-attachments/assets/751cbc67-efc7-4628-9d5f-e613ce118313)

**Comment by appflowy** [comment]
Hi @Twilightssuperb . No problem, I will check it. Thanks for your logs

**Comment by Twilightssuperb** [comment]
Any news? Sorry for tricky troubleshooting, IDK how it happened.

Should I'd better backup my data folder, delete it from main location and restore DB from file (https://docs.appflowy.io/docs/appflowy/product/data-storage#restore-specific-local-user-data-manually)?

**Comment by Twilightssuperb** [comment]
Well, seems data folder is damaged. I backuped it and tried to make clear AppFlowy instance, then pressed "Import data", but got error. Then I tried to edit .db-files (`cache.db` and `flow-database-db`), but blank page at start appeared again.

@appflowy, I attached data ZIP backup [there](https://dropmefiles.com/XzkTy) with no password etc., please, mention me after downloading, I will delete it. IDK what to do now, 'cause there are some important notes in my DB, it's hard to lost it. Sorry for incovenience.

**Comment by annieappflowy** [comment]
@Twilightssuperb , we'll take a look and reach out tomorrow

**Comment by Twilightssuperb** [comment]
@appflowy, try password `123`.

**Comment by Twilightssuperb** [comment]
No reply month later... C'mon, please, help me restore my database anyway, at least as DB-file, it doesn't matter, I still have no access to some important notes from AppFlowy. I know, it's my big fault and I shouldn't launch versions >0.4.9 on Win 8.1, but... 😢

**Comment by candideu** [comment]
@Twilightssuperb are you using the Cloud sync, by any chance?

**Comment by Twilightssuperb** [comment]
@candideu, nope, only local account. My data was damaged by my ridiculous mistake.

**Comment by candideu** [comment]
That's too bad... I faced a similar issue with Windows 10 (see: https://github.com/AppFlowy-IO/AppFlowy/issues/6979#issuecomment-2571743828) was only able to recover my data because it was backed up using the free sync account.

I've faced the same issue with another local-only app a while back (Focalboard), where the app wouldn't load my data after an upgrade. The devs did their best, but were unable to assist. Since then, I try to use local-apps with cloud sync features (or File History, if it's markdown based).

What I *did* manage to do with that other app is open my db file in a SQLite editor like [SQLiteStudio](https://sqlitestudio.pl/) and [DB Browser for SQLite](https://sqlitebrowser.org/). I was able to export the table which contained my data as a .csv, then imported it into Airtable. It was challenging, but I was able to salvage my data.

Appflowy's data structure seems to be a little more nebulous, though.... I have zero clue where my notes are stored on the computer, and which app I could use to open and recover them.... 
- https://github.com/AppFlowy-IO/AppFlowy/discussions/455
- https://github.com/AppFlowy-IO/AppFlowy/issues/2504#issuecomment-1543623178
- https://github.com/AppFlowy-IO/AppFlowy/issues/3334
- https://www.reddit.com/r/AppFlowy/comments/1ewcifu/ensuring_data_integrity/
- Tried reading through this doc page, but it didn't offer much guidance: https://docs.appflowy.io/docs/appflowy/product/data-storage

**Comment by khorshuheng** [comment]
@Twilightssuperb Sorry for the delay, @appflowy is on leave at the moment so I can't reach him immediately to get the current status.

Apologize again for the inconvenience, could you send us the file again? I will get some other colleague/myself to see if we can recover the database.

**Comment by Twilightssuperb** [comment]
@khorshuheng, https://dropmefiles.com/tpCS7 default password is `123`

**Comment by khorshuheng** [comment]
@Twilightssuperb Thanks. I have downloaded the file, you may remove the link now.

**Comment by khorshuheng** [comment]
Based on the file, it appears that the RocksDB data is corrupted - the .sst file is missing. I will research if there are ways to repair the database.

**Comment by Twilightssuperb** [comment]
@khorshuheng, should I search on PC and attach another files? How to find this manifest file, e.g.?

**Comment by khorshuheng** [comment]
@Twilightssuperb While the manifest file exists inside collab_db folder, it can't be used for the data in backup. I tried that, hoping that it would work, but failed.

In any case, I can find the historical document updates in the .sst files in collab_db_backup, and is able to dump the content using the sst_dump tools. Though I will still need to figure out how to read the data via API and reconstruct the document in JSON, if possible. Will need to consult my other colleagues on this.

**Comment by Twilightssuperb** [comment]
I can attach something from `C:\users\[...]\AppData\Roaming\io.appflowy` backup, but I'm not sure that it could clear.

**Comment by Twilightssuperb** [comment]
@khorshuheng, main notes language is Russian, so it must be in Unicode. Well, I don't have Discord account, so I'll think of it.

**Comment by khorshuheng** [comment]
Noted. Is there any other way that you would prefer me to send you the data, if not by discord? Since the content is now in plaintext, it's probably not a good idea to have public download link to your data in this issue.
